website/docs/developer/development-guide/code/versioning-releases.md

```diff
@@ -0,0 +1,118 @@
+
+```
+    git tag -a vx.y.z -m "Release x.y.z"
+    git push origin vx.y.z
+```
+
+4. **Running the deployment**
+
+Publishing of new releases is handled by a Bamboo release plan and is manually
+triggered.
+
+If you created a new release plan in step one, you will need to create a new
+bamboo deployment plan.
+
+**Creating a Bamboo deployment plan**
+
+* In the ORCA project (https://ci.earthdata.nasa.gov/browse/ORCA-OI), click
+  `Actions -> Configure Plan` at the top right.
+* Scroll to the bottom of the branch list in the bottom left and select
+  `Create Plan Branch`.
+* Add the values in that list. Choose a display name that makes it very clear 
+  this is a deployment branch plan. Release (branch name) seems to work well. 
+  Make sure you enter the correct branch name.
+* Important Deselect Enable Branch - if you do not do this, it will immediately
+  fire off a build.
+* Do Immediately On the Branch Details page, enable 'Change Trigger'. Set the 
+  Trigger type to manual, and this will prevent commits to the branch from 
+  triggering the build plan. You should have been redirected to the 'Branch 
+  Details' tab after creating the plan. If not, navigate to the branch from
+  the list where you clicked 'Create Plan Branch' in the previous step.
+* Go to the Variables tab. Ensure that you are on your branch plan and not the
+  master plan: You should not see a large list of configured variables, but 
+  instead a dropdown allowing you to select variables to override, and the tab 
+  title will be Branch Variables. Set a DEPLOYMENT variable appropriate for the
+  release (defaults to last committer). This should be cumulus-from-npm-tf 
+  except in special cases such as incompatible backport branches. Then set:
+  * ORCA_VERSION: `<version number>`
+  * RELEASE_FLAG: true
+* Enable the branch from the 'Branch Details' page.
+* Run the branch using the 'Run' button in the top right.
+
+Bamboo will build and run unit tests against that tagged release.
+
+**Create a new ORCA release on github**
+
+The release is automated in Bamboo, but the step must be manually started. If
+you set the `RELEASE_FLAG` to `true` and the build steps passed, you will
+be able to run the manual 'Release' step in Bamboo.
+
+The CI release scripts will create a release based on the release version tag,
+as well as uploading release artifacts to the Github release for the Terraform
+modules provided by Cumulus. The Terraform release artifacts include:
+
+* A multi-module Terraform .zip artifact containing filtered copies of the 
+  tf-modules, packages, and tasks directories for use as Terraform module sources.
+
+Just make sure to verify the appropriate .zip files are present on Github after
+the release process is complete.
+
+**Merge the base branch back into develop and master**
+
+Finally, you need to merge the version update changes back into develop and 
+master.
+
+If this is the latest version, you can simply create a PR to merge the release 

```

Comment:
In the vein of 'super-nitpicky', it'll actually be two PRs instead of 'a PR'.

Reply:
Updated to:  If this is the latest version, create the PRs to merge the release branch 
into develop and master.